src/main/java/uk/gov/companieshouse/model/dto/email/PendingPaymentEmailData.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.companieshouse.model.dto.email;
+
+public class PendingPaymentEmailData extends EmailData {
+    private String to;
+    private String subject;
```

Comment:
Please rebase master - `to` and `subject` have been moved into the superclass and do not need to be redeclared here, along with their getters and setters